target/linux/generic/backport-5.15/850-v5.17-0002-PCI-pci-bridge-emul-Add-definitions-for-missing-capa.patch

```diff
@@ -21,9 +21,9 @@ Signed-off-by: Lorenzo Pieralisi <lorenzo.pieralisi@arm.com>
 
 --- a/drivers/pci/pci-bridge-emul.c
 +++ b/drivers/pci/pci-bridge-emul.c
-@@ -270,6 +270,49 @@ struct pci_bridge_reg_behavior pcie_cap_
- 		.ro = GENMASK(15, 0) | PCI_EXP_RTSTA_PENDING,
- 		.w1c = PCI_EXP_RTSTA_PME,
+@@ -313,6 +313,49 @@ struct pci_bridge_reg_behavior pcie_cap_
```

Comment:
Please remove this patch, it was applied upstream, see https://github.com/gregkh/linux/commit/739c052b9b26bc5abff3f6a5139292c00bc1964e